README.md

```diff
@@ -235,9 +242,9 @@ $ yarn --silent beehive-flow status
     "major": 0,
     "minor": 11,
     "patch": 0,
-    "preRelease": "alpha"
+    "preRelease": "rc"
   },
-  "versionString": "0.11.0-alpha",
+  "versionString": "0.11.0-rc",
   "branchType": "feature",
   "branchState": "feature",
   "isLatestReleaseBranch": false
```

Comment:
Should this be "isLatest" as you use that in the Jenkinsfile example?

Reply:
Yep. Fixed.